src/app/modules/group/components/member-list/member-list.component.ts

```diff
@@ -32,9 +33,15 @@ const descendantUsersColumns: Column[] = [
   { field: 'parentGroups', header: 'Parent group(s)' },
 ];
 
+const descendantTeamsColumns: Column[] = [
+  { field: 'name', header: 'Name', sortable: true },
+  { field: 'parentGroups', header: 'Parent group(s)' },
+  { field: 'parentGroups', header: 'Parent group(s)' },
```

Comment:
wrong field name?
I suppose that's because you haven't tested as you did not have teams. In such case, ask me to update the db (not always possible but may be very simple).
Here I have added teams in 164594055934439342 (which is a child of 947670356997220543) that can be managed by the usual user.

Reply:
fixed